crates/router/src/core/payment_methods/cards.rs

```diff
@@ -131,6 +131,9 @@ pub async fn create_payment_method(
     storage_scheme: MerchantStorageScheme,
     payment_method_billing_address: Option<Encryption>,
     card_scheme: Option<String>,
+    network_token_requestor_reference_id: Option<String>,
+    network_token_locker_id: Option<String>,
+    network_token_payment_method_data: Option<Encryption>,
```

Comment:
Should be make a struct and make the struct option and inside the struct all these are mandatory? By doing this avoid handling multiple option case evaluation?